lib/vagrant-r10k/action/deploy.rb

```diff
@@ -58,8 +62,8 @@ def deploy(env, config)
           runner = R10K::TaskRunner.new([])
```

Comment:
This class doesn't seem to exist anymore. It seems that, despite what the spec tests say, this code doesn't run at all.

I'm going to have to do some work on it, but it appears the internals of R10K have changed a lot more than we thought...